libraries/AP_DDS/README.md

```diff
@@ -272,6 +276,51 @@ or
 std_srvs.srv.Trigger_Response(success=True, message='Vehicle is Armable')
 ```
 
+Call the prearm check service:
+
+```bash
+$ ros2 service call /ap/prearm_check std_srvs/srv/Trigger
+requester: making request: std_srvs.srv.Trigger_Request()
+
+response:
+std_srvs.srv.Trigger_Response(success=False, message='Vehicle is Not Armable')
+
+or
+
+std_srvs.srv.Trigger_Response(success=True, message='Vehicle is Armable')
+```
+
+Clear the Rally Points list (the provided Point will be ignored):
+
+```bash
+ros2 service call /ap/rally_set ardupilot_msgs/srv/RallySet "{clear: true}"
+requester: making request: ardupilot_msgs.srv.RallySet_Request(rally=ardupilot_msgs.msg.Rally(point=geographic_msgs.msg.GeoPoint(latitude=0.0, longitude=0.0, altitude=0.0), break_altitude=0.0, land_dir=0.0, flag_favorable_winds=False, flag_do_auto_land=False, altitude_frame=0), clear=True)
+
+response:
+ardupilot_msgs.srv.RallySet_Response(success=True, size=0)
+```
+
+Append a Rally Point:
```

Comment:
It would probably be good to show fractional lat/long and be clear that the altitude is meters.

Reply:
Do we have to show units of measure when in SI?

Reply:
I think so because unfortunately some ArduPilot metrics are not SI (it's a work in progress).

Reply:
I think this README needs an update now that you removed some of the flags and optional features.

Regarding units, the DDS interface follows ROS REP-103. Units are SI unless otherwise stated.

Reply:
@Ryanf55 the Readme has been updated.

Reply:
@timtuxworth Readme has been updated with fractional lat/lon